uint16 TYPE_ID = 1
geometry_msgs/PoseStamped start_point
geometry_msgs/PoseStamped centre
uint16 angle